test/internal/log/.gitignore

```diff
@@ -0,0 +1 @@
+*.log
```

Comment:
Rubocop only checks ruby files and this repo isn't copped. But it can be. New PR.